apps/dashboard/src/components/workflow-editor/steps/email/maily-config.tsx

```diff
@@ -127,8 +128,23 @@ export const createExtensions = (props: {
           as: 'div',
         });
       },
+      addAttributes() {
+        const attributes = this.parent?.();
+        return {
+          ...attributes,
+          aliasFor: {
+            default: null,
+          },
+        };
+      },
```

Comment:
This allows preserving the `aliasFor`, otherwise the TipTap editor would ignore the property when inserting a new node.